app/client/src/components/shared/LocationSearch.js

```diff
@@ -165,126 +167,205 @@ function LocationSearch({ route, label }: Props) {
+              'Addr_type',
+            ],
+            name: 'ArcGIS',
+          },
+        ],
+      },
+      {
+        type: 'group',
+        name: 'EPA Tribal Areas',
+        placeholder: 'Search EPA tribal areas...',
+        sources: [
+          {
+            layer: new FeatureLayer({
+              url: `${services.data.tribal}/1`,
+              listMode: 'hide',
+            }),
+            searchFields: ['TRIBE_NAME'],
+            suggestionTemplate: '{TRIBE_NAME}',
+            exactMatch: false,
+            outFields: ['TRIBE_NAME'],
+            name: 'EPA Tribal Areas - Alaska Native Villages',
+          },
+          {
+            layer: new FeatureLayer({
+              url: `${services.data.tribal}/2`,
+              listMode: 'hide',
+            }),
+            searchFields: ['TRIBE_NAME'],
+            suggestionTemplate: '{TRIBE_NAME}',
+            exactMatch: false,
+            outFields: ['TRIBE_NAME'],
+            name: 'EPA Tribal Areas - American Indian Reservations',
+          },
+          {
+            layer: new FeatureLayer({
+              url: `${services.data.tribal}/3`,
+              listMode: 'hide',
+            }),
+            searchFields: ['TRIBE_NAME'],
+            suggestionTemplate: '{TRIBE_NAME}',
+            exactMatch: false,
+            outFields: ['TRIBE_NAME'],
+            name: 'EPA Tribal Areas - American Indian Off-Reservation Trust Lands',
+          },
+          {
+            layer: new FeatureLayer({
+              url: `${services.data.tribal}/4`,
+              listMode: 'hide',
+            }),
+            searchFields: ['TRIBE_NAME'],
+            suggestionTemplate: '{TRIBE_NAME}',
+            exactMatch: false,
+            outFields: ['TRIBE_NAME'],
+            name: 'EPA Tribal Areas - American Indian Oklahoma Statistical Areas',
+          },
+          {
+            layer: new FeatureLayer({
+              url: `${services.data.tribal}/5`,
+              listMode: 'hide',
+            }),
+            searchFields: ['TRIBE_NAME'],
+            suggestionTemplate: '{TRIBE_NAME}',
+            exactMatch: false,
+            outFields: ['TRIBE_NAME'],
+            name: 'Virginia Federally Recognized Tribes',
+          },
+        ],
+      },
+      {
+        type: 'layer',
+        name: 'Watershed',
+        placeholder: 'Search watersheds...',
+        sources: [
+          {
+            layer: new FeatureLayer({
+              url: services.data.wbdUnconstrained,
+              listMode: 'hide',
+            }),
+            searchFields: ['name', 'huc12'],
+            suggestionTemplate: '{name} ({huc12})',
+            exactMatch: false,
+            outFields: ['name', 'huc12'],
+            name: 'Watersheds',
+          },
+        ],
+      },
+      {
+        type: 'webservice',
+        name: 'Monitoring Location',
+        menuHeaderExtra:
+          '(Below items open into the Monitoring Report page in a new tab)',
```

Comment:
```suggestion
          '(Below items open into the Monitoring Report page in a new browser tab)',
```

I think this text should include `browser` so that it is consistent with all of our `(opens new browser tab)` disclaimers.

Reply:
Good call, that is more consistent!